_posts/2017-07-05-SpringIO17-Summary.md

```diff
@@ -0,0 +1,773 @@
+---
+layout: post
+authors: [dieter_hubau, gina_de_beukelaer, hans_michiels, jeff_mesens]
+title: 'Spring I/O 2017 Recap'
+image: /img/springio2017.jpg
+tags: [Spring, IO, Pivotal, Google, Spring Boot, Java, Reactive, Reactor, DDD, CQRS, Spring Cloud, Spring Cloud Stream, Serverless]
+category: Spring
+comments: true
+---
+
+> On the 18th and 19th of May, we had another great edition of Spring I/O, brought to us by organizer [Sergi Almar](https://twitter.com/sergialmar){:target="_blank"}.
+In this blog post, we will go over some of our favourite sessions of the conference.
+
+> **DISCLAIMER**: this is obviously a very opinionated list of talks. This does not mean that the other talks were of lower quality, it just means we were not able to attend or did not take extensive notes.
```

Comment:
Sounds pretty negative. How about this?

"We were not able to attend all talks and therefore some pieces of this post are opinionated."